### DISCUSSION TITLE
Is it possible to move unfinished plotting to another hard drive?

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#1982] [Q&A]
Sometimes, in order to optimize the plots after already started, is it possible to move the uncompleted plots to another hard drive? If I do need to start all over, do I have to manually delete all the uncompleted plots?

**Comment by cccat6** [comment]
It is not possible to move the unfinished plots to the other drive. If you stop the plotting in the process. The program will not automatically delete, so you have to do it manually.
